content/blog/2018/egress-monitoring-access-control.md

```diff
@@ -0,0 +1,546 @@
+    200
+    200
+    200
+    ```
+
+    Access to all the topics of _edition.cnn.com_ is allowed.
+
+1.  Let's examine the Mixer log and see that no more requests with `sourceNamespace` equal `"politics"` appear in the log.
+
+    ```command
+    $ kubectl -n istio-system logs $(kubectl -n istio-system get pods -l istio-mixer-type=telemetry -o jsonpath='{.items[0].metadata.name}') mixer | grep egress-access | grep cnn
+    ```
+
+### Dashboard
+
+As an additional security measure, let our organization's operation people visually monitor egress traffic.
+
+1.  Follow the steps 1-3 of the [Visualizing Metrics with Grafana](/docs/tasks/telemetry/using-istio-dashboard/#viewing-the-istio-dashboard) task.
+
+1.  Send requests to _cnn.com_ from `$SOURCE_POD`:
+
+    ```command
+    $ kubectl exec -it $SOURCE_POD -c sleep -- bash -c 'curl -sL -o /dev/null -w "%{http_code}\n" http://edition.cnn.com/politics; curl -sL -o /dev/null -w "%{http_code}\n" http://edition.cnn.com/sport; curl -sL -o /dev/null -w "%{http_code}\n" http://edition.cnn.com/health'
+    404
+    200
+    200
+    ```
+
+    Since `$SOURCE_POD` is in the `default` namespace, access to  [edition.cnn.com/politics](https://edition.cnn.com/politics) is forbidden, as previously.
+
+1.  Send requests to _cnn.com_ from `$SOURCE_POD_IN_POLITICS`:
+
+    ```command
+    $ kubectl exec -it $SOURCE_POD_IN_POLITICS -n politics -c sleep -- bash -c 'curl -sL -o /dev/null -w "%{http_code}\n" http://edition.cnn.com/politics; curl -sL -o /dev/null -w "%{http_code}\n" http://edition.cnn.com/sport; curl -sL -o /dev/null -w "%{http_code}\n" http://edition.cnn.com/health'
+    200
+    200
+    200
+    ```
+
+1.  Scroll the dashboard to _HTTP services_, _istio-egressgateway.istio-system.svc.cluster.local_ section. You should see a graph similar to the following:
+
+    {{< image width="100%" ratio="19.47%"
+    link="../img/dashboard-egress-gateway.png"
+    caption="Dashboard section of istio-egressgateway"
+    >}}
+
+    We can see the _404_ error code received by the _sleep_ application from the _default_ namespace, _unknown_ version, in the _Requests by Source, Version and Response Code_ section on the left. This information can give the operation people a visual clue regarding which application tries to perform forbidden access. We can also see the _200_ code received by _sleep_ applications from the _default_ and _politics_ namespaces, so we can know which applications performed valid access to external services.
+
+## Comparison with HTTPS egress traffic control
+
+In this use case the applications used HTTP and Istio Egress Gateway performed TLS origination for them. Compare this approach with HTTPS originated by the applications, by issuing HTTPS requests to _edition.cnn.com_.
```

Comment:
Instead of:

Compare this approach ...

How about:

Alternatively, the applications could originate TLS themselves by issuing HTTPS requests to _edition.cnn.com_.